packages/nodes-base/nodes/Zoom/Zoom.node.ts

```diff
@@ -0,0 +1,834 @@
+import {
+	IExecuteFunctions,
+} from 'n8n-core';
+import {
+	IDataObject,
+	INodeExecutionData,
+	INodeType,
+	ILoadOptionsFunctions,
+	INodeTypeDescription,
+	INodePropertyOptions,
+} from 'n8n-workflow';
+import {
+	zoomApiRequest,
+	zoomApiRequestAllItems,
+} from './GenericFunctions';
+
+import {
+	meetingOperations,
+	meetingFields,
+} from './MeetingDescription';
+
+import {
+	meetingRegistrantOperations,
+	meetingRegistrantFields,
+
+} from './MeetingRegistrantDescription';
+
+import {
+	webinarOperations,
+	webinarFields,
+} from './WebinarDescription';
+import * as moment from 'moment-timezone';
```

Comment:
If you're using a single resource from a lib, I think it's best to import only that resource.